src/hu_nmt/data_augmentator/dependency_parsers/dependency_parser_factory.py

```diff
@@ -15,8 +15,24 @@ class DependencyParserFactory:
         'fr': lambda: StanzaDependencyParser(lang='fr', processors='tokenize, mwt, pos, lemma, depparse'),
         'hu': lambda: SpacyDependencyParser(lang='hu'),
     }
+
+    tokenizers = {
+        'en': lambda: StanzaDependencyParser(lang='en', processors='tokenize, pos'),
+        'ja': lambda: StanzaDependencyParser(lang='ja', processors='tokenize, pos'),
+        'hr': lambda: StanzaDependencyParser(lang='hr', processors='tokenize, pos'),
+        'ru': lambda: StanzaDependencyParser(lang='ru', processors='tokenize, pos'),
+        'cs': lambda: StanzaDependencyParser(lang='cs', processors='tokenize, mwt, pos'),
+        'de': lambda: StanzaDependencyParser(lang='de', processors='tokenize, mwt, pos'),
+        'uk': lambda: StanzaDependencyParser(lang='uk', processors='tokenize, mwt, pos'),
+        'fr': lambda: StanzaDependencyParser(lang='fr', processors='tokenize, mwt, pos'),
```

Comment:
do we need to load the POS taggers? isn't the tokenizer enough?

Reply:
I wanted to count the words in the sentence and with Stanza there is no other way to identify punctuation as far as I know.